src/extensions/views/ViewManifestExtension/components/AppFrame/useAppActions.ts

```diff
@@ -28,7 +28,8 @@ export const useAppActions = (
     versions,
   );
   const { handle: handlePermissionRequest } = AppActionsHandler.useHandlePermissionRequest(appId);
-  const { handle: handleAppFormUpdate } = AppActionsHandler.useHandleAppFormUpdate();
+  // Note: handleAppFormUpdate removed as formPayloadUpdate action is no longer in @saleor/app-sdk/app-bridge
+  // const { handle: handleAppFormUpdate } = AppActionsHandler.useHandleAppFormUpdate();
```

Comment:
it is, you just don't have latest node modules

please run `pnpm install` and revert these changes